tests/conftest.py

```diff
@@ -1739,7 +1739,16 @@ def rgw_obj_fixture(request):
     rgw_deployments = get_deployments_having_label(
         label=constants.RGW_APP_LABEL, namespace=config.ENV_DATA["cluster_namespace"]
     )
-    if rgw_deployments:
+    try:
+        storageclass = OCP(
```

Comment:
[Constructor of OCP](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ocp.py#L42) doesn't call any oc command. There should be added: `storageclass.get()` at the end of `try` block.

Reply:
Right! Missed it. Thanks